web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :username, :string
+    field :profile_image_url, :string
+    timestamps()
+  end
+
+  @doc """
```

Comment:
Since this is `WIP` still, I reserver the right to say "I was totally going to delete this"